GliaWidgets/Sources/Component/AttachmentList/Item/AttachmentSourceItemStyle.swift

```diff
@@ -90,3 +90,38 @@ extension AttachmentSourceItemStyle {
         lhs.accessibility == rhs.accessibility
     }
 }
+
```

Comment:
The same

Reply:
fixed [f3566d1](https://github.com/salemove/ios-sdk-widgets/pull/845/commits/f3566d1688549ebaf323e9078b956ebe650b0270)